### PR TITLE
Remove Blackmnyna, Chikka, Junebug, Twitter legacy, old Zenvia channe…

### PR DESCRIPTION
**Pull request by norkans7**
…l types

### CODE REVIEW

**Review by rowanseymour — APPROVED** [review]

File: temba/channels/models.py [hunk 1]
```diff
@@ -130,7 +130,7 @@ def get_urls(self):
         """
         if self.claim_view:
             return [self.get_claim_url()]
-        else:
+        else:  # pragma: no cover
```
Comment (norkans7) [inline temba/channels/models.py:133]: Only channel types that was using this are CK and TT removed in this PR. all other types have the `claim_view` set.

Could as well remove the line

Comment (rowanseymour) [reply temba/channels/models.py:133]: let's just have this method be `return [self.get_claim_url()]`

**Comment by rowanseymour** [comment]
@norkans7 weird merge going on here

**Comment by norkans7** [comment]
I just addressed that

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**